CMakeLists.txt

```diff
@@ -393,8 +395,6 @@ endif()
 if(OCOS_ENABLE_MATH)
   target_include_directories(ocos_operators PUBLIC ${dlib_SOURCE_DIR})
   list(APPEND OCOS_COMPILE_DEFINITIONS ENABLE_MATH)
-
-  # The dlib matrix implementation is all in the headers, no library compiling needed.
```

Comment:
Is this comment no longer relevant? 

FWIW I find explanatory comments like this to be very helpful.

Reply:
agree, but this information causes misleading since the one source file actually added in the above.